site/content/en/docs/manual/basics/workspace.md

```diff
@@ -1,38 +1,122 @@
+| 2       | [Controls sidebar](/docs/manual/basics/controls-sidebar/)                                           |
+| 3       | [Objects sidebar](docs/manual/basics/objects-sidebar/)                                              |
+| 4       | Image annotation area, described in this article.                                                   |
 
-  ![](/images/image139_mapillary_vistas.jpg)
+See:
 
-- `Z-axis slider` - Allows you to switch annotation layers hiding the upper layers
-  (slider is enabled if several z layers are on a frame).
-  This element has a button for adding a new layer. When pressed, a new layer is added and switched to it.
-  You can move objects in layers using the `+` and `-` keys.
+- [Image settings in CVAT](#image-settings-in-cvat)
+  - [Adding grid overlay to image in CVAT](#adding-grid-overlay-to-image-in-cvat)
+  - [Changing color settings of image is CVAT](#changing-color-settings-of-image-is-cvat)
+- [Adding layers and Z-axis slider](#adding-layers-and-z-axis-slider)
+- [Interacting with Objects](#interacting-with-objects)
 
-  ![](/images/image140.jpg)
+## Image settings in CVAT
 
-- `Image settings panel` -  used to set up the grid and set up image brightness contrast saturation.
+The **Image settings** panel serves
+as a versatile tool for fine-tuning the visual aspects of your image.
+Whether you need to brighten the image,
+increase contrast, or make other adjustments, this panel is your go-to.
 
-  - Show `Grid`, change grid size, choose color and transparency:
+Additionally, the panel allows you
+to overlay a grid on the image for more precise annotation.
 
-  ![](/images/image068_mapillary_vistas.jpg)
+By default, the **Image settings** panel is not visible. To access
+it, click on the **Arrow Up** (![Image Grid Icon](/images/image-grid-icon.jpg))
+icon located at the bottom of the workspace.
 
-  - Adjust `Brightness`/`Contrast`/`Saturation` of too exposed or too
-    dark images using color settings (it affects only how a user sees the image, not the image itself).
+![Image quality panel](/images/image-quality-panel.jpg)
 
-  ![](/images/image164_mapillary_vistas.jpg)
+### Adding grid overlay to image in CVAT
 
-  - `Reset color settings` to default values.
+To add the grid to the image, do the following:
+
+1. Open the **Image Settings** panel.
+2. Locate and check the box that
+   allows you to overlay a grid on the image.
+3. Specify the grid cell size in square millimeters
+   by entering the desired number in the **Size** field.
+4. From the **Color** drop-down list,
+   select the color of the grid.
+5. Use the Opacity slider to change the
+   transparency of the grid overlay.
+
+### Changing color settings of image is CVAT
+
+To change the color setting of the image is CVAT, do the following:
+
+1. Open the **Image Settings** panel.
+2. Use the slider to change the color quality.
+
+There are four color quality settings in CVAT:
```

Comment:
Please, add a note that these settings affect only how users see image, not the image itself

Reply:
Maybe we should put that note in `Image settings` section